src/app/learn/get-started/basics/serve-project/ios-simulator/index.html

```diff
@@ -0,0 +1,103 @@
+<stache
+  navOrder="3"
+  pageTitle="Serve to an iOS simulator">
+
+  <stache-page-summary>
+    <p>
+      To serve your SPA to an iOS simulator and debug it, follow these instructions.
+    </p>
+  </stache-page-summary>
+
+  <ol>
+    <li>
+      <p>
+        Download and install XCode. This install can take a long time.
+      </p>
+    </li>
+    <li>
+      <p>
+        Open XCode and select <strong>XCode</strong> in the toolbar.
+      </p>
+    </li>
+    <li>
+      <p>
+        Navigate to Open Developer Tool and select <strong>Simulator</strong>. This opens the iOS simulator to simulate a full iOS device.
+      </p>
+    </li>
+    <li>
+      <p>
+        Select the simulated power button to turn on the simulator.
+      </p>
+    </li>
+    <li>
+      <p>
+        Set up the simulated device to allow local development.
+      </p>
+      <ul>
+        <li>
+          <p>
+            On your iOS device, navigate to <strong>Settings</strong>, <strong>Safari</strong> and turn off <strong>Prevent Cross-Site Tracking</strong>.
+          </p>
+        </li>
+        <li>
+          <p>
+            On your Mac, open <strong>Keychain Access</strong> to extract the SKY UX certificate from your machine for use in iOS.
+          </p>
+        </li>
+        <li>
+          <p>
+            Select the SKYUX-Developer-CA certificate and then navigate to <strong>File</strong>, <strong>Export Items</strong>.
+          </p>
+        </li>
+        <li>
+          <p>
+            Save the certificate to your preferred location.
+          </p>
+        </li>
+        <li>
+          <p>
+            Drag the <sky-code>.cer</sky-code> file to the simulator. No prompt appears.
+          </p>
+        </li>
+      </ul>
+    </li>
+    <li>
+      <p>
+        Run <sky-code>skyux serve --browser safari</sky-code> to serve your SPA to Safari, and then use the link provided in the command line to view your SPA.
+      </p>
+    </li>
+    <li>
+      <p>
+        To allow console logging and general debugging, follow these steps:
+      </p>
+      <ul>
+        <li>
+          <p>
+            On your Mac, open Safari and navigate in the toolbar to <strong>Safari</strong>, <strong>Preferences</strong>, <strong>Advanced</strong> and enable <strong>Show Develop menu in menu bar</strong>.
+          </p>
+        </li>
+        <li>
+          <p>
+            Open <a href="http://preview.mobify.com">http://preview.mobify.com</a> in the iOS simulator.
```

Comment:
Is this mobify step necessary? Why couldn't you just link directly to the SPA URL in the iOS toolbar?

Reply:
I have no idea. This is the guidance in the instructions, but I don't have a Mac to test this. Do you want me to change this? And if so, does the next bullet need to change too?

Reply:
I removed the mobify steps.